{{ cookiecutter.__project_name_kebab }}/pyproject.toml

```diff
@@ -27,30 +27,27 @@ classifiers = [
     "Operating System :: OS Independent",
     "Programming Language :: Python",
     "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.7",
     "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Framework :: Django",
-    "Framework :: Django :: 3.0",
-    "Framework :: Django :: 3.1",
     "Framework :: Django :: 3.2",
-    "Framework :: Django :: 4.0",
     "Framework :: Django :: 4.1",
+    "Framework :: Django :: 4.2",
     "Framework :: Wagtail",
-    "Framework :: Wagtail :: 2",
-    "Framework :: Wagtail :: 4",
+    "Framework :: Wagtail :: 4.1",
+    "Framework :: Wagtail :: 4.2",
+    "Framework :: Wagtail :: 5.0",
 ]
 dynamic = ["version"]
 dependencies = [
-    "Django>=3.0,<4.2",
```

Comment:
I think we should retain at least an explicit minimum version constraint on django, even if we move to an open-ended range. @zerolab - do you have opinions on this one?

Reply:
I agree on the minimum version constraint suggestion